### DISCUSSION TITLE
Hey I was thinking something...

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1834] [General]
Hey I was wondering is it possible to make fnf available for all device like android,ios,ipad,pc if it’s possible tell me if it isn’t possible maybe in the future it will work

**Comment by steviegt6** [comment]
Possible? Sure. Mobile (Android, specifically) as already done. iOS is possible but a release would be tough given Apple's App Store regulations. iOS and iPad are one and the same. PC already has access to FNF.
There'd need to be redesigns and special cases for different platforms, though!

**Reply by ghost** [reply]
Ok

**Reply by suprstarrd** [reply]
Super sorry to necropost, but another option is [AltStore](https://altstore.io)! The only disadvantage is it requires a computer, and at that point, you should probably just play the game on desktop.

Ninja did say that the full game is coming to mobile, but you probably already know that.

**Comment by libauror-a** [comment]
And also a mobile port goal was reached on the kickstarter

**Comment by ninjamuffin99** [comment]
would come to mobile yes
